plugins/source/gandi/codegen/main.go

```diff
@@ -0,0 +1,109 @@
+package main
+
+import (
+	"bytes"
+	"embed"
+	"fmt"
+	"go/format"
+	"log"
+	"os"
+	"path"
+	"runtime"
+	"strings"
+	"text/template"
+
+	"github.com/cloudquery/cloudquery/plugins/source/gandi/codegen/recipes"
+	"github.com/cloudquery/plugin-sdk/codegen"
+)
+
+//go:embed templates/*.go.tpl
+var templatesFS embed.FS
+
+func main() {
+	var resources []*recipes.Resource
+	resources = append(resources, recipes.CertificateResources()...)
+	resources = append(resources, recipes.DomainResources()...)
+	resources = append(resources, recipes.LiveDNSResources()...)
+	resources = append(resources, recipes.SimpleHostingResources()...)
+
+	for _, r := range resources {
+		r.Infer()
+	}
+	if err := recipes.SetParentChildRelationships(resources); err != nil {
```

Comment:
Unless we have resolver/mock templates that handle parent & child relations in this plugin (do we?), we can probably remove this call and the associated function.

Reply:
It's used to properly generate child table names, completely optional, but gandi doesn't have 100% of resources covered (or the sdk has 100% implemented) so more resources will pop up and this could help us down the line (we'll eventually have more then a few child tables)